MDANSE/Src/MDANSE/Framework/Jobs/IJob.py

```diff
@@ -45,6 +45,13 @@
 RUNSCRIPT = """\
 #!{executable}
 
+import os
+os.environ['OMP_NUM_THREADS'] = '1'
+os.environ['OPENBLAS_NUM_THREADS'] = '1'
+os.environ['MKL_NUM_THREADS'] = '1'
+os.environ['VECLIB_MAXIMUM_THREADS'] = '1'
+os.environ['NUMEXPR_NUM_THREADS'] = '1'
```

Comment:
Consider:
```suggestion
os.environ.update(
    OMP_NUM_THREADS="1",
    OPENBLAS_NUM_THREADS ="1",
    MKL_NUM_THREADS="1",
    VECLIB_MAXIMUM_THREADS="1",
    NUMEXPR_NUM_THREADS="1",
)
```
to avoid repetition.

Reply:
updated